spec/chunky_png/datastream_spec.rb

```diff
@@ -58,4 +59,76 @@
       expect{physical_chunk.dpiy}.to raise_error(ChunkyPNG::UnitsUnknown)
     end
   end
+
+  describe '#iTXt_chunk' do
+    it 'should read iTXt chunks correctly' do
+      filename = resource_file('itxt_chunk.png')
+      ds = ChunkyPNG::Datastream.from_file(filename)
+      int_text_chunks = ds.chunks.select { |chunk|
+        chunk.is_a?(ChunkyPNG::Chunk::InternationalText)
+      }
+      expect(int_text_chunks.length).to eq(2)
+
+
+      coach_uk = int_text_chunks.find { |chunk|
+        chunk.language_tag == 'en-gb'
+      }
+      coach_us = int_text_chunks.find { |chunk|
+        chunk.language_tag == 'en-us'
+      }
```

Comment:
Style nit pick: I prefer `do ... end` if the block takes up multiple lines like here (or use `{...}` if it all fits on a single line.